3741/3/views/layout.erb

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```

Comment:
ERB is a preprocessor not type of file, so file should has name like `my_awesom_page.html.erb`

Reply:
delete doctype line